packages/nimble-components/src/select/index.ts

```diff
@@ -199,6 +199,23 @@ export class Select
         return !(this.multiple || typeof this.size === 'number');
     }
 
+    /** @internal */
+    public labelSlot!: HTMLSlotElement;
+
+    /** @internal */
+    @volatile
+    public get labelContent(): string {
```

Comment:
Why doesn't the `Combobox` need this same implementation and set of tests?

Reply:
As discussed in the other comment threads, combobox doesn't need `aria-label` set to resolve its label accessibility violation, so it doesn't need `labelContent` (only used to set `aria-label`).